pyproject.toml

```diff
@@ -147,7 +147,7 @@ disallow_untyped_decorators = false # mypy gets confused by pytest decorators
 [tool.pytest.ini_options]
 asyncio_mode = "auto"
 asyncio_default_fixture_loop_scope = "session"
-timeout = 30
+timeout = 120
```

Comment:
One of the tradeoffs of switching to Mistral API is that their service in my experience was less reliable, there were times where there was a larger initial lag, so I bumped up this value to be more accomodating.